Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocAutoCompleteReferenceValue.css

```diff
@@ -1,248 +1,146 @@
+  gap: var(--remotion-themed-spacing);
+  align-items: center;
 }
 
-span.bocAutoCompleteReferenceValue.disabled span.bocAutoCompleteReferenceValueButton
+.bocAutoCompleteReferenceValue span.body img
 {
-  border: 1px solid #BCBCBC;
+  /* currently used for overriding img.Icon rule in Style.css: */
+  vertical-align: middle;
+  border-style: none;
 }
 
-span.bocAutoCompleteReferenceValueButton:hover
+.bocAutoCompleteReferenceValue span.content,
+.bocAutoCompleteReferenceValue span.icon
 {
-  border: 1px solid #7CB2E6;
+  grid-row: 1;
+  grid-column: 1;
 }
 
-span.bocAutoCompleteReferenceValueButton
+.bocAutoCompleteReferenceValue span.content
 {
-  background-color: #EAEEFE;
+  grid-column: icon / content;
 }
 
-span.bocAutoCompleteReferenceValueButton img
+.bocAutoCompleteReferenceValue span.content span.bocAutoCompleteReferenceValueInput
 {
+  position: absolute;
   height: 100%;
-  width: 15px;
-  border: none;
-  -moz-box-sizing: border-box;
-  -webkit-box-sizing: border-box;
-  box-sizing: border-box;
-  background-image: url('../Image/sprite.svg#AutoCompleteReferenceValueDropDownButtonFace');
-  background-size: 16px 16px;
-  background-position: 0 center; /*-1px for border*/
-  background-repeat: no-repeat;
 }
 
-span.bocAutoCompleteReferenceValue.disabled span.bocAutoCompleteReferenceValueButton
+.bocAutoCompleteReferenceValue span.content span.bocAutoCompleteReferenceValueInput input,
+.bocAutoCompleteReferenceValue span.content.hasIcon span.bocAutoCompleteReferenceValueInput input
 {
-  background-color: #EFEFF0;
+  box-sizing: border-box;
+  height: 100%;
 }
 
-span.bocAutoCompleteReferenceValue.disabled span.bocAutoCompleteReferenceValueButton img
+.bocAutoCompleteReferenceValue span.content.hasIcon span.bocAutoCompleteReferenceValueInput input
 {
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
+  width: calc(var(--remotion-themed-control-width) + var(--default-icon-size) + var(--remotion-themed-spacing));
+  padding-left: calc(var(--default-icon-size) + 2 * var(--remotion-themed-spacing));
+  padding-right: calc(var(--default-icon-size) + var(--remotion-themed-spacing-small));
 }
 
-span.bocAutoCompleteReferenceValueButton img:hover
+.bocAutoCompleteReferenceValue span.content span.bocAutoCompleteReferenceValueInput input
 {
-  background-color: #DEECFC;
+  width: calc(var(--remotion-themed-control-width) + var(--default-icon-size) + var(--remotion-themed-spacing));
+  padding-left: var(--remotion-themed-spacing);
+  padding-right: calc(var(--default-icon-size) + var(--remotion-themed-spacing-small));
 }
 
-span.bocAutoCompleteReferenceValueButton img:active
+.bocAutoCompleteReferenceValue span.icon img
 {
-  background-color: #C5E0FC;
+  position: relative;
+  width: var(--default-icon-size);
+  height: var(--default-icon-size);
+  padding-left: var(--remotion-themed-spacing);
+  z-index: 1;
```

Comment:
```suggestion
  z-index: --var(--z-index-max);
```

Reply:
In this case, `z-index` cannot be set to `--z-index-max` for the icon since it would then interfere with the drop down menu of the search results, as it would still be visible when the drop down menu is expanded.

Reply:
disclaimer: I only got the gist from the comments

I'm very careful about the z-index. using z-index may result in unwanted sideeffects when using popup panels and may result either in the z-index hiding the elemnt when it's used in a popup with a higher z-index or peeking through another overlay. -> I will need to really understand why there is no other way to do the layout.

Reply:
If the z-index of the icon is not higher than that of the input element, then the icon is not visible. This is because we wanted to overlay these two so that the icon is visible within the input on its left-hand side. Setting it to --z-index-max, though, would lead to other icons of input elements below the current one to be visible through the expanded drop down menu since then the drop down results and the input icon would have the same z-index. Thus, I set the input icon to 1 so that it is visible within the input element, but the drop down results still overlay the icon.

Reply:
We may need to move this into a live session, but I don't undestand why the icon is "before" / "lower" than the textbox given the icon is rendered / inserted after the textbox. So, there should be no need for z-index (as far as I can tell from this discussion).